src/MultiReport/Keyboard.h

```diff
@@ -67,8 +67,8 @@ class Keyboard_ {
     return HID().getLEDs();
   };
 
-  HID_KeyboardReport_Data_t report_;
-  HID_KeyboardReport_Data_t last_report_;
+  HID_KeyboardReport_Data_t pending_report_;
```

Comment:
Naming wise, "pending report" and "current report" sound like they're both "the report we're about to send" - Can you talk a bit about what you're trying to convey with the names?

Reply:
I tried to describe it in the commit message, but here goes: I think of the "current report" as "the report that is currently in effect", and the "pending report" is "the report that hasn't been sent yet". Since the host keeps the last report it received in effect, the "current" one keeps causing things to happen (key repeats) even if no new reports are being sent. That's why I think it makes sense. I'm not going to get hung up on this name change if you don't like it, though; that's why I spun it out into a separate commit.

Reply:
And now I see that second commit message. Sorry for missing it before. I understand why the names should change. Lemme ponder if there are other names that feel clearer.

Reply:
I may be ever so slightly biased toward those names because “pending” and “current” both have seven letters, making the two variables have the same length.

Reply:
I've been pondering this - Once it's been sent, it's not the current report. I'd rather get the functionality merged, so I'll cherry-pick the functionality change for now and we can leave this hanging around to ponder.

Reply:
I still say that it becomes the "current" report when it's sent to the host. It's the one that is _currently_ active on the host. The report that I've labeled "pending" really shouldn't be called "current", because most of the time when a plugin interacts with it, it's not even complete.

`last_report` & `next_report` might be alternatives that you're happier with. I don't really like using "last report" to describe the first one, for reasons stated earlier, but that's only really relevant inside the `sendReport()` function. I maintain that "current" and "pending" are more accurately descriptive names, but "last" and "next" are also okay, in my opinion. And there may be better options, of course.